.github/workflows/ci.yml

```diff
@@ -10,17 +10,46 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental }}
```

Comment:
If we’re keeping `|| ${{ matrix.experimental }}`, should we remove this?

Reply:
removed.